plugins/cmd/chainlink-solana/main.go

```diff
@@ -70,9 +79,18 @@ func (c *chainPlugin) NewRelayer(ctx context.Context, config string, keystore lo
 		return nil, fmt.Errorf("failed to create chain: %w", err)
 	}
 	r := pkgsol.NewRelayer(c.lggr, chainSet)
+
+	err = c.promServer.Start()
+	if err != nil {
+		err = fmt.Errorf("Failed to start prometheus server: %w", err)
+		c.lggr.Critical(err.Error())
+		return nil, err
+	}
```

Comment:
So for now we are running multiple chain instances inside of each plugin process, so this method can be called more than once. IIUC `func main()` seems right though :shrug: - and we can include this is some kind of more general LOOPP wrapper later on if the redundant main code keeps growing.